### PR TITLE
Build(index.html): Shows empty-page with blue background

### PR DESCRIPTION
**Pull request by ZJaf**
This is just the landing page with the blue background

### CODE REVIEW

**Comment by RichardLitt** [comment]
Great. Why did you make this PR?

**Comment by ZJaf** [comment]
Addressing issue

**Comment by RichardLitt** [comment]
Same as the previous issue: can you rename this PR to something that describes what it does, not just what it is?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.DS_Store
+node_modules
```
Comment (RichardLitt) [inline .gitignore]: Shouldn't need node_modules in this PR, because you haven't added any node modules.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <!-- Required meta tags -->
+    <title>Independent bookstores</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- CSS Ref -->
+    <link rel="stylesheet" type="text/css" href="assets/stylesheets/main.css">
+
+    <!-- Bootstrap Ref-->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">
```
Comment (RichardLitt) [inline index.html]: Don't need bootstrap, Font Awesome, or jQuery for this PR. This is extra code.

**Comment by RichardLitt** [comment]
Closing in favor of #34, which solves this issue. Thank you for the work! Hope it was helpful. More coming.

**Pull request closed without merging**